recipe/meta.yaml

```
@@ -10,60 +10,71 @@ source:
   sha256: d3e8aa2e1f4092dfe8158523a4c9508f0759a400e8cca8a711669361ce23e883
 
 build:
+  number: 0
+  # No dependencies for py39
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - glue = glue_qt.main:main
-  script: {{ PYTHON }} -m pip install . -vv
-  string: win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [win]
-  string: unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [unix]
-  number: 1
-  osx_is_app: true  # [osx]
```

```suggestion
  osx_is_app: true  # [osx]
```
Probably still want this. Does the app launch on your laptop without it?
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-app